modules/metastorage/src/main/java/org/apache/ignite/internal/metastorage/impl/MetaStorageManagerImpl.java

```diff
@@ -133,32 +145,38 @@ public class MetaStorageManagerImpl implements MetaStorageManager, MetastorageGr
 
     private final MetaStorageMetricSource metaStorageMetricSource;
 
+    private final MetastorageRepairStorage metastorageRepairStorage;
+    private final MetastorageRepair metastorageRepair;
```

Comment:
```suggestion
    private final MetastorageRepairStorage metastorageRepairStorage;
    
    private final MetastorageRepair metastorageRepair;
```

Reply:
These two objects work together; currently, there is a nice grouping, but if I add that blank like, the grouping will just go away. We should not just blindly separate each field from others, I think.